### PR TITLE
OSOE-158: Typo fixes

### PR DESCRIPTION
**Pull request by Luko6**
OSOE-158

### CODE REVIEW

File: Lombiq.VueJs.Samples/Views/VueSfc/Index.cshtml [hunk 1]
```diff
@@ -9,7 +9,7 @@
    the .vue file's name without the extension. This is used to find and render the template shape.
 
    This tag helper renders the template as a shape (in this case "demo-sfc" becomes VueComponent-DemoShape) and injects
-   the script which was tanspiled using Rollup and Bublé among others.
+   the script which was tanspiled using Rollup and Buble among others.
```
Comment (Piedone) [inline Lombiq.VueJs.Samples/Views/VueSfc/Index.cshtml]: Bublé is the name.

File: Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 const { readdirSync } = require('fs');
 
-const asDirent = { withFileTypes: true };
```
Comment (Piedone) [inline Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js]: This looks more like an abbreviation for "as directory entry". Check if indeed, and use a non-abbreviated name then. Same for `dirent` below.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 function readDirectoryFunction(filter) {
-    return (rootPath) => readdirSync(rootPath, asDirect).filter(filter).map((direct) => direct.name);
+    return (rootPath) => readdirSync(rootPath, asDirent).filter(filter).map((dirent) => dirent.name);
 }
 
 module.exports = {
-    getVueApps: readDirectoryFunction((direct) => direct.isDirectory()),
-    getVueComponents: readDirectoryFunction((direct) => direct.name.endsWith('.vue') && direct.isFile()),
+    getVueApps: readDirectoryFunction((dirent) => dirent.isDirectory()),
+    getVueComponents: readDirectoryFunction((dirent) => dirent.name.endsWith('.vue') && dirent.isFile()),
```
Comment (Piedone) [inline Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js:11]: As I've written under https://github.com/Lombiq/Orchard-Vue.js/pull/60#discussion_r1000697996:

>  Check if indeed, and use **a non-abbreviated name then**.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js (hunk 3 above)
Comment (Luko6) [reply Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js:11]: [Dirent](https://nodejs.org/api/fs.html#class-fsdirent) is a name used by the library though, so I think it's most logical that this stays as is.

Comment (Piedone) [reply Lombiq.VueJs/Assets/Scripts/helpers/get-vue-files.js:11]: I see, OK then.

**Review by Piedone — APPROVED** [review]

**Pull request merged**